lab-claudia/ng-template/app/entry.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// Build sass - gets custom bootstrap and theme
+require('./scss/main.scss');
+
+// Require node modules
+const path = require('path');
+
+// Require npm modules
+const angular = require('angular'); //create angular module
+const camelcase = require('camelcase');
+const pascalcase = require('pascalcase');
+
+// Require angular modules
+// Injected in modules to do routing and bootstrap
+const ngTouch = require('angular-touch'); // Dependency of bootstrap
+const ngAnimate = require('angular-animate'); // Dependency of bootstrap
+const uiRouter = require('angular-ui-router');
+const uiBootstrap = require('angular-ui-bootstrap'); // Need touch and animate to use
+
+// Create angular module
+// Add everything we need to module
+const demoApp = angular.module('demoApp', [ngTouch, ngAnimate, uiRouter, uiBootstrap]);
+
+// WHAT DOES REQUIRE.CONTEXT DO?
```

Comment:
@loomnugget great tutorial comments - keep this up - also, keep it on a separate branch...  `master` should remain relatively comment free and production ready